db/migrate/20200910154901_add_user_reference.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserReference < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :comments, :user,
+    add_column :users, :email, :string
```

Comment:
## MIGRATION ERROR
The issue over here is that your line number 3 ends with a ','(comma) if you remove this, your migration should work with no issue. but I saw that below, you added again the email column to the user table. so you must delete the last file added to your migration that only contain the email